Library/Formula/connect.rb

```diff
@@ -0,0 +1,17 @@
+class Connect < Formula
+  desc "provides SOCKS and HTTPS proxy support to SSH."
+  homepage "https://bitbucket.org/gotoh/connect/wiki/Home"
```

Comment:
While no official notice to state that it is official in all aspects it appears so.
- same bitbucket username (gotoh) as original meadowy.org/~gotoh (that no longer exists)
- the [initial commit](https://bitbucket.org/gotoh/connect/commits/7c036cbffb61822f129754336b1046bf658894d7) suggests this was a migration due to a disk issue
- used by debian for their official [.deb ](https://packages.qa.debian.org/c/connect-proxy.html)